### PR TITLE
T009 nav desktop

### PR DESCRIPTION
**Pull request by AwesomeJackify**

### CODE REVIEW

File: src/components/Navbar.tsx [hunk 1]
```diff
@@ -26,29 +27,51 @@ function NavBar() {
...
+            <a href="/">Community</a>
+            <a href="/">Join Us</a>
+          </div>
+					{/* <div className="socials_wrapper">
+						<SocialIcon url="https://www.facebook.com/ytechnz/" />
+						<SocialIcon url="https://www.instagram.com/ytechnz/" />
+					</div>	 */}
+					{/* <a href="/" className="button header__cta hide-for-mobile">
+						CTA button
+					</a> */}
```
Comment (KennethZhu904) [inline src/components/Navbar.tsx]: remove those comments if you are not using this code anymore

File: src/screens/Landing.tsx [hunk 2]
```diff
@@ -2,21 +2,42 @@ import Footer from '../components/Footer';
...
+                <h1>Image</h1>
+                <Image src="https://drive.google.com/file/d/1GD4DUn3ehKOqwz32R-tMjGnWSZw6wVlG/view" />
             </Section>
+
+            {/*Video*/}
+            <Section>
+                <h1>YouTube Video</h1>
+                <YouTubeVideo src="https://www.youtube.com/watch?v=iy3VW4L0dAY" />
+            </Section>
+
```
Comment (KennethZhu904) [inline src/screens/Landing.tsx]: remove that space

File: src/styles/navbar.scss [hunk 3]
```diff
@@ -1,6 +1,16 @@
...
+// .socials_wrapper {
+//     justify-self: center;
+
+//     a {
+//         height: 40px !important;
+//         width: 40px !important; 
+//         margin-right: 10px;
+//     }
+// }
+
```
Comment (KennethZhu904) [inline src/styles/navbar.scss]: remove those comments if you are not using it

File: src/widgets/Image.tsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+  //  Obtain google drive image ID. 
+  var id = parts[parts.length-2];
+
+  //  Construct the image URL for website display. 
+  var output = 'https://drive.google.com/uc?export=view&id=' + id;
+
+  return <img className="cust_image" alt="YTech NZ drive" src={output}/>
+}
+
```
Comment (KennethZhu904) [inline src/widgets/Image.tsx:18]: remove that space

File: src/components/Navbar.tsx [hunk 5]
```diff
@@ -1,6 +1,7 @@
 import React, { useState } from 'react';
-import phLogo from '../assets/logo.png'
+import phLogo from '../assets/logo_rgb.png'
 import '../styles/navbar.scss'
+import { SocialIcon } from 'react-social-icons';
```
Comment (KennethZhu904) [inline src/components/Navbar.tsx]: remove any unnecessary imports, especially SocialIcon. SocialIcon is not being used in this code.

**Review by KennethZhu904 — CHANGES_REQUESTED** [review]
Overall, great effort on this ticket.  Just a couple of comments that I have made in this PR. Please make sure to make the changes before tonight's meeting.

**Review by KennethZhu904 — APPROVED** [review]
LGTM!

**Review by johnchen383 — CHANGES_REQUESTED** [review]
Remove package-lock.json please

**Review by johnchen383 — APPROVED** [review]
Some refactoring could be done to clean the codebase but I think this should be fine for now. LGTM if ur fine with the edits I've made.

**Pull request merged**